src/commands/pools/delete_map.ts

```diff
@@ -16,36 +16,48 @@ export default {
 
     options: [
         {
-            name: "map_uuid",
-            description: "The uuid of the map to remove",
-            type: ApplicationCommandOptionTypes.INTEGER,
+            name: "map_uuids",
+            description: "Comma separated list of uuids of the map to remove",
+            type: ApplicationCommandOptionTypes.STRING,
             required: true,
         },
     ],
 
-    callback: async ({interaction}) => {
-        const {options, channelId, guildId, member} = interaction;
+    callback: async ({ interaction }) => {
+        const { options, channelId, guildId, member } = interaction;
 
         // ensure that the command is being run in a server
         if (!channelId || !guildId) {
             return "This command can only be run in a text channel in a server";
         }
 
-        let guild = await Guild.findOneBy({id: guildId});
+        let guild = await Guild.findOneBy({ id: guildId });
         if (!guild) {
             guild = new Guild(guildId);
         }
-        if(!isAdmin(member as GuildMember, guild)) {
+        if (!isAdmin(member as GuildMember, guild)) {
             await interaction.reply({
                 ephemeral: true,
                 content: "Only admins are allowed to use this command"
             })
             return;
         }
+        await interaction.deferReply();
 
         // get the command parameters
-        const mapUuid = ensure(options.getInteger("map_uuid"));
+        const mapUuids = ensure(options.getString("map_uuids")).split(",");
+        const deleteGameResponses: string[] = [];
+        const deleteGamePromises = [];
+
+        for (const mapUuid of mapUuids) {
+            deleteGamePromises.push(deleteGameMap(parseInt(mapUuid), guildId)
```

Comment:
input `1,2s,3` will cause exception here, catch and push response 'map id 2s is invalid'

Reply:
Done